lib/build.js

```diff
@@ -2,7 +2,13 @@ const path = require("path")
 const findUp = require("find-up")
 const { init } = require("./init")
 
-const build = async ({ dir, verbose, skipNextBuild }) => {
+const build = async ({
+  dir,
+  verbose,
+  skipBuild,
```

Comment:
This is a breaking change right? `skipNextBuild => skipBuild`. We will need to keep the old option and deprecate it first.

Reply:
I added it as deprecated 
<img width="1208" alt="image" src="https://github.com/seamapi/nextjs-server-modules/assets/11433064/abd33e6c-3513-4c52-9526-71745e19258b">

Reply:
This option needs to be retained and deprecated here as well, or it's still a breaking change: https://github.com/seamapi/nextjs-server-modules/pull/47/files#diff-1ae50dc56d6f821fbc0183fc424426e7cf420c9f5e0a0a5fd0e1801a573b8fd0R18